docs/source/zh/serialization.md

```diff
@@ -0,0 +1,181 @@
+<!--Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 导出为 ONNX
+
+在生产环境中部署 🤗 Transformers 模型通常需要或者能够受益于，将模型导出为可在专门的运行时和硬件上加载和执行的序列化格式。
+
+🤗 Optimum 是 Transformers 的扩展，可以通过其 `exporters` 模块将模型从 PyTorch 或 TensorFlow 导出为 ONNX 及 TFLite 等序列化格式。🤗 Optimum 还提供了一套性能优化工具，可以在目标硬件上以最高效率训练和运行模型。
+
+本指南演示了如何使用 🤗 Optimum 将 🤗 Transformers 模型导出为 ONNX。有关将模型导出为 TFLite 的指南，请参考[导出为 TFLite 页面](tflite)。
```

Comment:
Should we maybe link to the English version of Export to TFLite for now since this doc doesn't exist in Chinese?

Reply:
> Should we maybe link to the English version of Export to TFLite for now since this doc doesn't exist in Chinese?

Hi @stevhliu I think for this document it's fine to keep the Chinese version since I have make the translations for tflite and will commit soon.

For future docs, I'm wondering if there's any helper to identify if the document is referenced in the translated documents? If not, it will be difficult to maintain from the translators' sides to do it manually when the number of zh docs grow. It will definately take efforts when there're dozens of links in one doc like this. 

![image](https://github.com/huggingface/transformers/assets/76979429/25e103a7-f4c5-4175-91c6-ff91a7e4c3a3)


Otherwise, the redirect message seems fine with me.
![image](https://github.com/huggingface/transformers/assets/76979429/bb5b49c4-2d6d-4c03-b11f-5094d23ca1c8)

Best

Reply:
I don't think we currently have any tools that do this. The example you referenced below is auto-generated so you won't have to touch those. Besides the task pages, I don't think you'll find too many other docs with as many links as that though. 🙂